keanu-project/src/test/java/io/improbable/keanu/vertices/intgr/IntegerVertexTest.java

```diff
@@ -172,4 +178,17 @@ public void canFindMinAndMax() {
         assertArrayEquals(new int[]{1, 2, 1, 4}, min.getValue().asFlatIntegerArray());
         assertArrayEquals(new int[]{2, 4, 3, 5}, max.getValue().asFlatIntegerArray());
     }
+
+    public static double calculateMeanOfVertex(IntegerVertex vertex) {
```

Comment:
Why are these helper methods in this class instead of `IntegerIfVertexTest`?

Reply:
I thought it might be a more useful place for them. You and Wes seem to agree that this should be moved, so I'll put them with the test class.